### ISSUE TITLE
Path parsing problem in Windows 10

### ISSUE THREAD

**Issue by hsmyers** [simeg/eureka#46]

**Comment by hsmyers** [comment]
There are two mistakes in get_if_available() that lead to a panic re unable to find git. The first is an incorrect choice in path.split. A windows path uses the semi-colon to divide concatenated paths, not a colon. The colon would be used at the beginning of the path. Second, the folder separator would be an escaped backslash, not a single forward-slash. One might insert the necessary '.exe' after the second bracket pair depending on the function call—I didn't look.
Here is a test version: The original could be used for all other OSs accordingly.

```
use std::env;
use std::fs;

fn main() {
    pub fn get_if_available(program: &str) -> Option<String> {
        if let Ok(path) = env::var("PATH") {
            for p in path.split(";") {
                let p_str = format!("{}\\{}.exe", p, program);
                println!("{:?}", p_str);
                if fs::metadata(p_str).is_ok() {
                    return Some(String::from(program));
                }
            }
        }
        None
    }

    println!("{:?}", get_if_available("git"));
}

```

**Comment by simeg** [comment]
Thanks for reporting. Fixed in #47. Will release new version later today.
